backend/service/aws/aws.go

```diff
@@ -263,6 +269,8 @@ func (r *regionalClient) S3() *s3.Client {
 	return r.s3.(*s3.Client)
 }
 
+func (r *regionalClient) S3Control() *s3control.Client { return r.s3control.(*s3control.Client) }
+
```

Comment:
We already expose all clients via the `GetDirectClient()` on the aws client struct, we dont need to expose them individually.

Reply:
Ah! Thanks for pointing, will update.